bundles/org.openhab.binding.goecharger/README.md

```diff
@@ -44,6 +44,7 @@ Currently available channels are
 | sessionChargeEnergyLimit | Number:Energy            | Wallbox stops charging after defined value, disable with 0    | 1 (r/w), 2 (r/w)  |
 | sessionChargedEnergy     | Number:Energy            | Amount of energy that has been charged in this session        | 1 (r), 2 (r)      |
 | totalChargedEnergy       | Number:Energy            | Amount of energy that has been charged since installation     | 1 (r), 2 (r)      |
+| transaction              | Number                   | 0 if no card, otherwise card ID                               | 2 (r/w)           |
```

Comment:
In code it seems to be UNDEF (as opposed to 0) of there is no transaction/card? Which also seems cleaner.

Reply:
The charger keeps this UNDEF in case no authorization happened yet. If the App was used to unlock, it's 0. If an RFID key was used, it's the number of the key. It's sensible to me this way, because of the difference between 0 and UNDEF.